CHANGES.txt

```diff
@@ -1,7 +1,18 @@
-1.2.0 (unreleased)
+#########
+CHANGELOG
+#########
+
+
+1.2.0 (2016-01-18)
 ==================
 
-- Nothing changed yet.
+- Adding the ability to define services imperatively. (#335)
+- Clean cornice/statics/ files. (#345)
+
+Bug fixes
+
+- Convert None to null before calling deserialize. (#342)
```

Comment:
Consider replacing the entry in the `CHANGES.txt` by "Convert `None` to `colander.null` before calling colander's `deserialize` function."